### ISSUE TITLE
Should we describe how to do a network based install?

### ISSUE THREAD

**Issue by bexelbie** [lukaskotek/fedora-handbook-doc#12]

**Comment by ghost** [comment]
I agree. I am planning to start writing of handbook's updated (extended) version soon. Install section needs to be updated including network based install option. Also adding solving common situations section is planned.
